src/MovieContainer.css

```diff
@@ -0,0 +1,5 @@
+.all-movie-cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  background-color: #e5e6eb;
```

Comment:
I am glad that we got some color and styling started. I do think that we will want to revisit the color scheme

Reply:
Yes, revisiting the color scheme is definitely a good idea!